docs/architecture/adr012-non-interactive-default-rules-for-zero-padding.md

```diff
@@ -0,0 +1,56 @@
+# ADR 012: Non-Interactive Default Rules for Zero Padding
```

Comment:
```suggestion
# ADR 013: Non-Interactive Default Rules for Zero Padding
```

because 012 has been taken. Ref: https://github.com/celestiaorg/celestia-app/tree/main/docs/architecture